docs/source/models/risk_effects/hemoglobin/index.rst

```diff
@@ -196,17 +189,21 @@ Relative risk values to calculate custom PAFs for maternal disorders can be acce
                location_id=1, # NOTE: you do not need to specify a location_id. Specifying any location ID will return results specific to location_id=1
                ) 
 
-Use the custom-calculated PAF values such that the maternal disorder incidence rate for an individual :math:`i` for a given affected maternal disorder subcause is as follows:
+Before being applied in the simulation, these relative risk values are first re-scaled to be relative to the TMREL exposure value of 120 g/L. We used linear interpolation to assign RR values for exposures between points provided by the GBD risk curve. For exposure values that fall outside of the range provided by the GBD risk curve, we assign the same RR value as the nearest point on the curve. `See the custom hemoglobin risk effect implementation used in the MNCNH simulation for specific details <https://github.com/ihmeuw/vivarium_gates_mncnh/blob/main/src/vivarium_gates_mncnh/components/hemoglobin.py>`__
 
-.. math::
+We calculate custom PAFs for hemoglobin on maternal disorders outcome for use in the MNCNH simulation. Code to generate these PAFs can be found here ``TODO: POST LINK``. We utilize the interactive context to initialize a population with assigned hemoglobin exposure values, assign relative risk values in accordance with observed exposure, and calculate PAFs among the population as (mean_rr - 1) / mean_rr among the age-stratified population. This PAF calculation process is limited in the following ways:
```

Comment:
Is this a TODO we need to address before merging? If not, can we specify the timeline, e.g. add a TODO box saying to fill in this link once the GBD 2023 update is complete?

Reply:
Yeah I'll add a note that we will calculate and add links once the 2023 update is complete!